src/triage/experiments/validate.py

```diff
@@ -815,13 +776,37 @@ def _run(self, scoring_config):
                         )
 
 
+class FeatureValidator(Validator):
+    def _run(self, feature_config):
+        feature_lookup = architect.feature_block_generators.FEATURE_BLOCK_GENERATOR_LOOKUP
+        available_keys = set(feature_lookup.keys())
+        given_keys = set(feature_config.keys())
+        bad_keys = given_keys - available_keys
+        if bad_keys:
+            raise ValueError(
+                dedent(
```

Comment:
`dedent`:
> Remove any common leading whitespace from every line in `text`

In other words, do these work?

I'm pretty sure that if you want to form a multiline, dedented message like this, it would have to be:

```
            raise ValueError(
                dedent(
                    """\
                    Section: features -
                    The following given feature types '{}' are unavailable.
                    Available metrics are: '{}'
                    """
                )
            )
```